src/test/java/com/wikia/webdriver/testcases/discussions/PollsTests.java

```diff
@@ -0,0 +1,160 @@
+package com.wikia.webdriver.testcases.discussions;
+
+import com.wikia.webdriver.common.contentpatterns.MercuryWikis;
+import com.wikia.webdriver.common.core.Assertion;
+import com.wikia.webdriver.common.core.annotations.Execute;
+import com.wikia.webdriver.common.core.annotations.InBrowser;
+import com.wikia.webdriver.common.core.drivers.Browser;
+import com.wikia.webdriver.common.core.helpers.Emulator;
+import com.wikia.webdriver.common.core.helpers.User;
+import com.wikia.webdriver.common.remote.discussions.DiscussionsClient;
+import com.wikia.webdriver.common.templates.NewTestTemplate;
+import com.wikia.webdriver.elements.mercury.components.discussions.common.*;
+import com.wikia.webdriver.elements.mercury.pages.discussions.PostsListPage;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+@Test(groups = "discussions-polls")
+public class PollsTests extends NewTestTemplate {
+
+    public static final int DEFAULT_ANSWERS_NUMBER = 2;
+
+    private void createInitialPostWithPoll() {
+        final PostEntity.Data data = DiscussionsClient.using(User.USER, driver).createPostWithUniqueData();
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "data" local variable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1481)